manifests/dev/vsphere-7.0/vanilla/vsphere-csi-driver.yaml

```diff
@@ -156,6 +156,9 @@ spec:
             - "--kube-api-qps=100"
             - "--kube-api-burst=100"
             - "--leader-election"
+            # Uncomment the line below to increase performance of online
+            # resize in appropriate vSphere versions
```

Comment:
I did not want to mention `vSphere 7.0U2 and above` here as this YAML exclusively pertains to 7.0 version. I am hoping that customers see this comment and switch to the appropriate versions where online resize is suuported.

Reply:
Since we are anyways checking at runtime whether online expand volume is supported or not, why do we have separate yaml (one per vSphere) for the same CSI driver?

Reply:
Specified the exact versions where online resize is supported